htdocs/core/ajax/constantonoff.php

```diff
@@ -71,7 +71,8 @@
 		if ($action == 'set') {
 			dolibarr_set_const($db, $name, $value, 'chaine', 0, '', $entity);
 		} elseif ($action == 'del') {
-			dolibarr_del_const($db, $name, $entity);
+			// We should save the constant as 0 instead of deleting the constant.
+			dolibarr_set_const($db, $name, 0, 'chaine', 0, '', $entity);
```

Comment:
I keep setting the value to '0' here because:
- lots of external modules will use this script
- the core module stock.php has a default const STOCK_ALLOW_NEGATIVE_TRANSFER set on 1, and uses ajax/constantonoff.php for this const.